static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -268,4 +268,9 @@
         }
     }
 
+    .media-primary & {
+        @include mq($from: mobile, $until: mobileLandscape) {
+            margin-top: -1rem;
```

Comment:
Could we just not add this padding instead? Maybe this should be rewritten to use modifier classes `--main-media` and `--in-body`. We should avoid nesting and overwriting styles when possible.